src/explore-education-statistics-admin/src/pages/release/pre-release/__tests__/PreReleasePageContainer.test.tsx

```diff
@@ -130,6 +130,47 @@ describe('PreReleasePageContainer', () => {
     });
   });
 
+  test('renders correctly when on release day but release is yet to be published', async () => {
+    const now = new Date(); // 2am on Publish Day
+
+    permissionService.getPreReleaseWindowStatus.mockResolvedValue({
+      access: 'Within',
+      start: subHours(now, 26),
+      scheduledPublishDate: subHours(now, 2),
+    });
+
+    preReleaseService.getPreReleaseSummary.mockResolvedValue(
+      testPreReleaseSummary,
+    );
+
+    renderPage();
+
+    await waitFor(() => {
+      const banner = within(screen.getByRole('region', { name: 'Contact' }));
+      expect(
+        banner.getByText('If you have an enquiry about this release contact:'),
+      ).toBeInTheDocument();
+
+      expect(banner.getByText('Test team:')).toBeInTheDocument();
+      expect(
+        banner.getByRole('link', { name: 'test@test.com' }),
+      ).toBeInTheDocument();
+
+      expect(screen.getByRole('link', { name: 'Content' })).toBeInTheDocument();
+
+      expect(
+        screen.queryByRole('heading', {
```

Comment:
Hey @amyb-hiveit do we need to adhere to some conventions about queryByRole usage in waitFors? Some performance implications?

Reply:
There's no reason this assertion should actually need to be within a `waitFor`. It should be possible to do the awaiting once to give it time to render, then make these assertions synchronously. I was just copying from the tests above in this case, so might give them the same treatment too :)